README.md

```diff
@@ -1,34 +1,54 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Next.js Starter ButterCMS
 
-## Getting Started
+Next.js starter project using ButterCMS. The project uses the sample content that is automatically created when you
```

Comment:
Maybe this is just me, but I wonder if maybe this would be a bit more effective if the relationship between the two (Butter and Next.js) were clearer here, as well as maybe giving us a quick sell on some of the features. I remember when trying butter out for the first time, I opted not to use the starter project because I wasn't sure what all it would give me over just trying to do it manually.

How do you feel about something like?

"This Next.js starter project fully integrates with dynamic sample content from your ButterCMS account, including menus(?), pages, blog posts, categories, and tags, all with a beautiful, custom theme with already-implemented search functionality. All of the included sample content is automatically created in your account dashboard when you sign up for a free trial of ButterCMS."

What do you think?

Reply:
Also maybe adding an additional paragraph. "Once created, this project can be easily and quickly deployed to Vercel from the CLI (see instructions below).